documentation/feature/weaponselect.md

```diff
@@ -5,30 +5,41 @@ group: feature
 parent: wiki
 ---
 
-## Overview
+## 1. Overview
 
-### Weapon select
-The number key can be used to quickly switch between weapons. (1 key - pistol, 2 key - rifle, 3 key - grenade launcher, 4 key - rocket launcher, 5 key - binocular)
+### 1.1 Holster weapon
+Adds the ability to holster a weapon on the back.
 
-### Holster weapon
-Adds the ability to holster a weapon on the back. (0 key)
+### 1.2 Quick weapon select
+Adds key bindings to quickly switch weapons while on foot or in a vehicle. (Unbound by default).
 
-### Engine select
-Quickly turn engine on and off (1 key - turn off, 2 key - turn on)
+### 1.3 Quick vehicle engine on/off
+Adds key bindings to quickly turn a vehicle engine on or off.
```

Comment:
I'm not sure if these keys are bound by default or not - if they are, they should be similar to how they are in other docs with <kbd>modifier</kbd>+<kbd>the_key</kbd> but if they're not, it should probably be uniform with the one you wrote above ("Unbound by default").